src/content/en/2021/caching.md

```diff
@@ -1,20 +1,488 @@
-featured_quote: TODO
-featured_stat_1: TODO
-featured_stat_label_1: TODO
-featured_stat_2: TODO
-featured_stat_label_2: TODO
-featured_stat_3: TODO
-featured_stat_label_3: TODO
+featured_quote: Adoption of cache headers continues to steadily grow as we learn to navigate the multiverse of caching choices. Optimize pages not only for HighEnd devices but also for the next billion users that access your product from LowEnd devices.
+featured_stat_1: 99.8%
+featured_stat_label_1: Pages caching fonts
+featured_stat_2: 51 trillion year
+featured_stat_label_2: Largest `max-age` value recorded
+featured_stat_3: 59.1%
+featured_stat_label_3: Mobile pages could save up to 1 MB with caching
 ---
 
-## TODO
+## Introduction
+
+Over the last two decades, the way we experience web applications has changed, giving us richer and more interactive content. But, unfortunately, this content comes with a cost in both data storage and bandwidth. Most of the time, this makes it harder for many of us to fully experience a web product when the network we use is degraded, or our device doesn't have enough space. Caching is both a solution to and the cause of some of these problems. Learning to navigate the multiverse of choices will enable you to build not only for *HighEnd* devices but also for the next billion users that access your product from *LowEnd* devices.
+
+Caching is a technique that enables the reuse of previously downloaded content, from simple static assets like Javascript, CSS files or basic string values to more complex JSON API responses.
+
+At its core, Caching avoids making specific HTTP requests and allows an application to feel more responsive and reliable to the user. Each request is usually cached in two main places:
+- **Content Delivery Network (CDN)**: Usually a third-party company like Cloudflare with the primary goal of replicating your data as closely as possible to where the user is accessing the application. Most CDNs have some default behavior, but mainly you can give them instructions on how to cache by using HTTP Headers.
+- **Browser**: Usually will either learn how to cache your resources internally or respect the HTTP Headers you defined to optimize the experience. On top of that, you will have access to additional manual caching strategies including storing simple strings in *Cookies*, complex API responses in *IndexedDB*, or entire JS or HTML resources in the *CacheStorage* with a *Service Worker*.
+
+In this chapter, we will mostly focus on the HTTP Headers used between the Browser and the CDN, briefly mentioning Service Worker caching strategies.
+
+## CDN Cache adoption
+
+A Content Delivery Network (CDN) is a group of servers spread out over several locations that store copies of data. This allows servers to fulfill requests based on the server closest to the end-user. Across the web in 2021, the most popular CDN used for Desktop was Cloudflare with 14% of total pages, followed by Google with 6%.
```

Comment:
A diagram here might help drive this point a bit further

Reply:
What kind of diagram are you thinking?

Reply:
Since we mention cloudflare, is it possible to use a diagram from their CDN content explanation? 
[CDN Diagram](https://www.cloudflare.com/img/learning/cdn/what-is-a-cdn/what-is-a-cdn.png)

Reply:
It's a nice diagram, but I'd be wary of leaning too much on Cloudflare docs (simply because we wouldn't want to be seen as favoring one CDN). Although they do have the best blogs and diagrams.
(As well as reviewing this chapter, I also work for Akamai, so maybe I'm biased!)

Reply:
Good point! Any suggestions on what kind of diagram we want to use then?